### DISCUSSION TITLE
密码和盐对了，加密方式也对了，但是无法解密之前用alist创建的文件

### DISCUSSION THREAD

**Discussion by SsuJojo** [AlistGo/alist#5157] [Invalid question]
重装系统前，在alist创建了加密空间，并上传了文件

换磁盘重装系统后，重新下载最新版alist并配置，发现无法解密crypt文件，表现为英文和拓展名解密成功，但数字和中文无法解密，显示乱码

希望好心人能帮助一下，文件重要

**Comment by anwen-anyi** [comment]
够呛啊，只能靠你自己穷举测试了吧。

-----

>表现为英文和拓展名解密成功，但数字和中文无法解密，显示乱码

这啥意思？没听懂

**Reply by SeanHeuc** [reply]
所有配置和原来的一样才行，你查查哪个配置错了吧，如果靠记忆的话人会记错的，如果是alist备份恢复了读不出来我可以看看

**Comment by SsuJojo** [comment]
> 够呛啊，只能靠你自己穷举测试了吧。
> 
> > 表现为英文和拓展名解密成功，但数字和中文无法解密，显示乱码
> 
> 这啥意思？没听懂

比如我一个文件名为file.zip是显示正常的，但是"文件.zip"只有".zip"是显示正常的

**Comment by foxxorcat** [comment]
详细描述一下你的使用环境。比如加密后上传的哪个驱动，操作系统环境，配置加密使使用的选项，上传下载用的alist版本。你不说清楚无法排查问题
而且你只说了文件名无法解密，那么文件数据是否可以解密。

**Reply by SsuJojo** [reply]
之前winodws系统现在linux，我怕是大小写敏感导致的

**Reply by SsuJojo** [reply]
数据无法解密

**Comment by anwen-anyi** [comment]
> 之前winodws系统现在linux，我怕是大小写敏感导致的

哪你可以在自己本地Windows搭建一下，试试能不能，所以在文档里面进行了两次提醒

你是不是也修改了文件名加密或者文件夹加密，这样的话估计更难了..

你的文件多大？不大的话 可以下载一个加密的文件到本地，在本地搭建一下把加密的也挂载上然后进行一次一次的尝试，每次保存配置后记得右下角刷新按钮刷新一下，在网盘测试有可能你密码对了 有缓存导致还没刷新过来 

在本地解密出来了再去网盘修改配置取回吧。本地存储挂载不用担心刷新的限制，也方便，只要配置正确了加密的文件就能使用了。
